setup.py

```diff
@@ -52,7 +52,12 @@ def read(fname):
                 'License :: OSI Approved :: BSD License',
                 ],
         entry_points={
-                'napari.plugin': ['zarpaint = zarpaint.plugin'],
-                'console_scripts': ['zarpaint = zarpaint._main:main']
-                },
+             "napari.manifest": [
+                 "zarpaint = zarpaint:napari.yaml",
+             ],
+             'console_scripts': [
+                 'zarpaint = zarpaint._main:main'
+            ]
+         },
+         package_data={"zarpaint": ["napari.yaml"]},
```

Comment:
Ah. Interesting — would you mind migrating this to setup.cfg while you're at it? The reason it's still setup.py is only historical.